package.json

```diff
@@ -23,16 +23,21 @@
     "gatsby-plugin-mdx": "^2.14.0",
     "gatsby-plugin-react-helmet": "^4.14.0",
     "gatsby-plugin-sitemap": "^4.10.0",
+    "gatsby-plugin-typescript": "^4.10.1",
```

Comment:
I don't think we need this plugin. TypeScript is supported out of the box now by default.